content/en/core/overview/architecture.md

```diff
@@ -8,31 +8,59 @@ description: >
+A NodeJS service which runs on the server and provides security and APIs for browsers and integrations. It also includes a custom implementation of filtered replication to allow it to support more concurrent users. See more at the [CHT Core API repo](https://github.com/medic/cht-core/tree/master/api) on Github.
 
-### sentinel
+#### sentinel
 
 Another NodeJS service running on the server, sentinel performs actions called transitions every time a document in CouchDB is added or modified. Some examples are validations, generating scheduled messages, automatic responses, creating patients, and sending alerts. See more at the [CHT Core Sentinel repo](https://github.com/medic/cht-core/tree/master/sentinel) on Github. 
 
-### PostgreSQL
+#### CouchDB
 
-A free and open source SQL database that we use for analytics queries for display in tools like klipfolio. We created a library called [couch2pg](https://github.com/medic/couch2pg) to replicate data from CouchDB into PostgreSQL. See more at the [PostgreSQL](https://www.postgresql.org) site.
+A free and open source NoSQL database used to store all data and configuration. This can be multiple instances clustered together for additional scalability. CouchDB is really good at replication which is the process of sending the data to another database and back again, which makes it ideal for replicating data to the phone for offline access. See more at the [CouchDB](http://couchdb.apache.org) site.
+
+#### nginx
+
+Provides SSL termination and routes requests to API.
+
+#### haproxy
+
+Provides audit logging for any request that makes it to CouchDB so any data access or modification can be validated at a later date.
+
+### CHT Sync
+
+The cht-sync product is a suite of tools for extracting and normalizing data from the Core Framework's CouchDB and rendering the data in analytics dashboards to measure the performance of the program. Read more detail in [cht-sync GitHub repository](https://github.com/medic/cht-sync).
+
+#### Logstash and PostgREST
+
+[Logstash](https://www.elastic.co/logstash/) streams data from CouchDB and forwards it to [PostgREST](https://postgrest.org/en/stable/) which provides REST endpoints to store the data in the PostreSQL database.
+
+#### PostgreSQL
+
+A free and open source SQL database used for analytics queries. See more at the [PostgreSQL](https://www.postgresql.org) site.
+
+#### DBT
+
+[DBT](https://www.getdbt.com/) is used to ingest raw JSON data from the postgres database and normalize it into a relational schema to make it much easier to query.
+
+#### Superset
+
+[Apache Superset](https://superset.apache.org/) is a free an open source platform for creating data dashboards.
 
 ## Client side
 
 ### cht-core
 
-[cht-core](https://github.com/medic/cht-core) is composed of the following web applications:
+[cht-core](https://github.com/medic/cht-core) is composed of two web applications:
```

Comment:
The `:` is hard to follow since you can't see the two web applications in a list. I'd suggest explain the two apps immediately to provide context, and linking down (actual links missing in the suggestion)

```suggestion
The CHT Core Framework provides two web applications, the [CHT app](#) for care teams and program staff, and an [app management interface](#) for program administrators. 
```

Reply:
Good call!